src/Microsoft.SqlTools.ServiceLayer/ShowPlan/ShowPlanGraphUtils.cs

```diff
@@ -50,19 +50,37 @@ private static ExecutionPlanEdges ConvertShowPlanEdgeToExecutionPlanEdge(Edge ed
             };
         }
 
-        private static List<ExecutionPlanGraphElementProperties> GetProperties(PropertyDescriptorCollection props)
+        private static List<ExecutionPlanGraphPropertyBase> GetProperties(PropertyDescriptorCollection props)
         {
-            List<ExecutionPlanGraphElementProperties> propsList = new List<ExecutionPlanGraphElementProperties>();
+            List<ExecutionPlanGraphPropertyBase> propsList = new List<ExecutionPlanGraphPropertyBase>();
             foreach (PropertyValue prop in props)
             {
-                propsList.Add(new ExecutionPlanGraphElementProperties()
+                var complexProperty = prop.Value as Microsoft.SqlTools.ServiceLayer.ShowPlan.ShowPlanGraph.ExpandableObjectWrapper;
```

Comment:
Can simplify type name to just `ExpandableObjectWrapper` because the namespace is already being declared on line 9.